memory/src/main/java/org/opensearch/ml/memory/index/ConversationMetaIndex.java

```diff
@@ -119,21 +117,29 @@ public void initConversationMetaIndexIfAbsent(ActionListener<Boolean> listener)
     /**
      * Adds a new conversation with the specified name to the index
      * @param name user-specified name of the conversation to be added
+     * @param applicationType the application type that creates this conversation
      * @param listener listener to wait for this to finish
      */
-    public void createConversation(String name, ActionListener<String> listener) {
+    public void createConversation(String name, String applicationType, ActionListener<String> listener) {
         initConversationMetaIndexIfAbsent(ActionListener.wrap(indexExists -> {
             if (indexExists) {
-                String userstr = userstr();
+                String userstr = client
+                    .threadPool()
+                    .getThreadContext()
+                    .getTransient(ConfigConstants.OPENSEARCH_SECURITY_USER_INFO_THREAD_CONTEXT);
                 IndexRequest request = Requests
                     .indexRequest(META_INDEX_NAME)
                     .source(
-                        ConversationalIndexConstants.META_CREATED_FIELD,
+                        ConversationalIndexConstants.META_CREATED_TIME_FIELD,
+                        Instant.now(),
```

Comment:
Since this is for create a new conversation, suggest create a variable to replace Line 134 and line 136 
```
Instant now = Instant.now();
```

Reply:
Good catch.